public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
Nora's asking if clicking the s-ref toggles the form appearing and dissappearing again. As it currently stands, clicking the link opens the form. I made a button in the form that closes the form, but clicking the link again does nothing. I'm going to try changing that now.